plugins/color_panel/gtk/color_panel_plugin.cc

```diff
@@ -0,0 +1,160 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include "include/color_panel/color_panel_plugin.h"
+
+#include <flutter_linux/flutter_linux.h>
+#include <gtk/gtk.h>
+
+// See color_panel.dart for documentation.
+const char kChannelName[] = "flutter/colorpanel";
+const char kShowColorPanelMethod[] = "ColorPanel.Show";
+const char kColorPanelShowAlpha[] = "ColorPanel.ShowAlpha";
+const char kHideColorPanelMethod[] = "ColorPanel.Hide";
+const char kColorSelectedCallbackMethod[] = "ColorPanel.ColorSelectedCallback";
+const char kClosedCallbackMethod[] = "ColorPanel.ClosedCallback";
+const char kColorComponentAlphaKey[] = "alpha";
+const char kColorComponentRedKey[] = "red";
+const char kColorComponentGreenKey[] = "green";
+const char kColorComponentBlueKey[] = "blue";
+
+static constexpr char kWindowTitle[] = "Flutter Color Picker";
+
+G_DECLARE_FINAL_TYPE(FlColorPanelPlugin, fl_color_panel_plugin, FL,
```

Comment:
I think we should put this in the header, per comments on the initial PR, even if there's nothing public on it at this point.